src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -227,7 +227,7 @@ public function showAllAction(Request $request, $page)
     public function showUnreadAction(Request $request, $page)
     {
         // load the quickstart if no entry in database
-        if ($page == 1 && $this->get('wallabag_core.entry_repository')->countAllEntriesByUsername($this->getUser()->getId()) == 0) {
+        if ($page == 1 && $this->get('wallabag_core.entry_repository')->countAllEntriesByUser($this->getUser()->getId()) == 0) {
```

Comment:
Hmm, we got tons of methods (especially in the repositories) where user means userid.